### PR TITLE
Goals

### PR DESCRIPTION
**Pull request by Ruzakiff**
make main menu
make basic game loop and player movement 
make player image

### CODE REVIEW

File: management/SpeedRun.md [hunk 1]
```diff
@@ -17,9 +17,9 @@
 1. 9/16 - 9/22
 	* Manager: Ryan Yang
 	* Goals:
-		1. GOAL1
-		1. GOAL2
-		1. GOAL3
+		1. Make Main Menu Screen.
+		1. Make Basic game loop, player movement
```
Comment (nfarnan) [inline management/SpeedRun.md:21]: Specifically, what movement will you implement this week?

File: management/SpeedRun.md [hunk 2]
```diff
@@ -17,9 +17,9 @@
...
+		1. Make Player Image
```
Comment (nfarnan) [inline management/SpeedRun.md:22]: And display it to the screen

**Review by nfarnan — CHANGES_REQUESTED** [review]

File: management/SpeedRun.md (hunk 1 above)
Comment (Ruzakiff) [reply management/SpeedRun.md:21]: forward, left/right

File: management/SpeedRun.md (hunk 2 above)
Comment (Ruzakiff) [reply management/SpeedRun.md:22]: yes

**Comment by cas380** [comment]
This week's goals got added to this pull request as commits, I can't open a new one.

1. Add blocks the player can collide with and fine-tune movement
2. Randomly generate some kind of path/obstacles out of the collide-able blocks (or placeholder blocks)
3. Create initial networking code, ping client to client with up to 4 players

**Review by cas380 — APPROVED** [review]

**Comment by nfarnan** [comment]
I see. We've got a couple of issues with this pull request. First, my requested edits for last week's goals were never made. Apparently I should have closed that pull request. Also, this PR includes changes to files throughout the repository. I'm not sure what you did to edit all of the files, but you'll need to rebase your repository on the current state of the organization repo and try again. Be sure that you are only changing the management file for your team.

For now, I'm closing this PR.

**Pull request closed without merging**